src/subsystems/nodejs/builders/strict-builder/nodejs_builder/nodejs_builder/lib/config.py

```diff
@@ -0,0 +1,6 @@
+from pathlib import Path
+from .module import env
+
+root = Path("/build")
+node_modules = root / Path("node_modules")
+bin_dir = node_modules / Path(".bin")
```

Comment:
If there is nothing of interest in `root` other than `node_modules`, then `root` should be removed and instead `node_modules` should default to `/build/node_modules`

Reply:
Refactored: node_modules is not its own derivation.
in that case `root=$out` saves copy time within the derivation.